### PR TITLE
Add apt-get downloader helper to fetch transitive dependencies

### PR DESCRIPTION
**Pull request by mrene**
Fetching .debs for transitive dependencies can get pretty tedious. There were a few stackoverflow posts about automating the process, this is a script that can be ran which fetches all dependencies using apt-get using raspbian's sources list.

It fetches a little more than necessary, but it's easy enough to cleanup.

Example:

```shell
$ ./download.sh libusb-dev
Downloading:
 libusb-dev
libusb-0.1-4
libc6-dev
libc6
libc-dev-bin
linux-libc-dev
libgcc1
gcc-8-base
Get:1 http://raspbian.freemirror.org/raspbian buster/main armhf gcc-8-base armhf 8.3.0-6+rpi1 [191 kB]
Get:2 http://raspbian.freemirror.org/raspbian buster/main armhf libc-dev-bin armhf 2.28-10+rpi1 [266 kB]
Get:3 http://raspbian.freemirror.org/raspbian buster/main armhf libc6 armhf 2.28-10+rpi1 [2,355 kB]
Get:4 http://raspbian.freemirror.org/raspbian buster/main armhf libc6-dev armhf 2.28-10+rpi1 [2,113 kB]
Get:5 http://raspbian.freemirror.org/raspbian buster/main armhf libgcc1 armhf 1:8.3.0-6+rpi1 [36.5 kB]
Get:6 http://raspbian.freemirror.org/raspbian buster/main armhf libusb-0.1-4 armhf 2:0.1.12-32 [21.6 kB]
Get:7 http://raspbian.freemirror.org/raspbian buster/main armhf libusb-dev armhf 2:0.1.12-32 [35.7 kB]
Get:8 http://raspbian.freemirror.org/raspbian buster/main armhf linux-libc-dev armhf 4.18.20-2+rpi1 [1,546 kB]
Fetched 6,565 kB in 3s (2,583 kB/s)
```

Sorta related to #18

### CODE REVIEW

**Comment by Ragnaroek** [comment]
Nice, thanks for the contribution!
One thing: How can I verify the public key? I did a shallow search, but only found this here https://archive.raspberrypi.org/debian/https://archive.raspberrypi.org/debian/raspberrypi.gpg.key and it is not the same as the one you checked-in.

Also: If you want you can add a short text to the main README file. I think it should be briefly mentioned there, so more people see that it exists.

**Comment by mrene** [comment]
I had exported it from a live install which had two keys (`gpg file.asc` will show you the contents). I removed it anyways, better to tell people to get them from their original locations instead of maintaining copies.

File: README.md [hunk 1]
```diff
@@ -52,7 +53,29 @@ into a folder on your host system and then mount this directory into your `docke
...
-If you do `apt-cache show libssl1.0.0` on the raspberry, you'll see this in the
+#### Using apt-get to fetch packages
+From a debian-based system, you may use apt-get in order to fetch packages directly from the raspbian archives. A script is provided for these 
+purposes in the `apt/` folder.
+
+Here is how you would download libssl1.0 and all of its dependencies. 
+```
+$ cd apt/
+# Install the raspbian keys into your local apt keychain
+$ sudo ./install-keys.sh
```
Comment (Ragnaroek) [inline README.md:64]: Is this file missing in the commit? (or am I missing something?)

**Review by Ragnaroek — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrene) [reply README.md:64]: indeed

**Review by Ragnaroek — APPROVED** [review]

**Pull request merged**